contracts/upgradeable_contracts/erc20_to_native/ForeignBridgeErcToNative.sol

```diff
@@ -157,6 +164,8 @@ contract ForeignBridgeErcToNative is BasicForeignBridge, ERC20Bridge, OtherSideB
 
         require(tokenBalance(fdToken).sub(curFDTokenBalance) == curHDTokenBalance);
 
+        mintRToken(_getTokenBalance());
```

Comment:
the oracles will call `swapTokens` till the moment when the Emergency Shutdown is executed. That's why I think the features "Two tokens support" and "Earning interest with locked DAI" should be independent: a new method to be called by oracles to convert Dai to rDai must be implemented, oracle must be changed to call this method from time to time, the monitor and the UI must be modified to operates with sum of balances Dai and rDai.

Reply:
So you suggest removing this call, right?

Reply:
yes